packages/artillery/lib/platform/aws-ecs/legacy/plugins/artillery-plugin-sqs-reporter/index.js

```diff
@@ -75,6 +75,54 @@ function ArtillerySQSPlugin(script, events) {
     });
   });
 
+  events.on('phaseStarted', (phaseContext) => {
```

Comment:
Is this code duplicated from somewhere else? if so would be useful to have a comment here to make refactoring it out later easier.